pkg/controller/networkpolicy/crd_utils_test.go

```diff
@@ -428,7 +428,7 @@ func TestToAntreaPeerForCRD(t *testing.T) {
 			_, npc := newController()
 			npc.addClusterGroup(&cgA)
 			npc.cgStore.Add(&cgA)
-			actualPeer := npc.toAntreaPeerForCRD(tt.inPeers, testCNPObj, tt.direction, tt.namedPortExists)
+			actualPeer := npc.toAntreaPeerForCRD(getAntreaClusterNetworkPolicyReference(testCNPObj), tt.inPeers, testCNPObj, tt.direction, tt.namedPortExists)
```

Comment:
I just thought about this: could we use `getACNPReference` / `getANPReference` as function names? It gets pretty long to read and we already use the ACNP / ANP abbreviations in other function names

Reply:
updated